.github/workflows/release.yml

```diff
@@ -20,6 +20,18 @@ jobs:
       - uses: sigstore/cosign-installer@v3
       - uses: fluxcd/flux2/action@main
       - uses: stefanprodan/timoni/actions/setup@main
+      - name: Setup Notation CLI
+        uses: notaryproject/notation-action/setup@v1
+        with:
+          version: "1.0.0"
```

Comment:
Should this be `1.1.0`?

Reply:
Yes. Resolved in [c7f9b52](https://github.com/stefanprodan/podinfo/pull/341/commits/c7f9b521fab56fe99ead827791d13ad2bc25ff2f).